.vscode/settings.json

```diff
@@ -1,12 +1,27 @@
 {
-  "editor.formatOnSave": true,
+  "[css]": {
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "editor.defaultFormatter": null,
+  "editor.formatOnSave": false,
+  "[html]": {
+    "editor.formatOnSave": false
+  },
   "files.encoding": "utf8",
   "files.eol": "\n",
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
+  "[javascript]": {
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
```

Comment:
It doesn't work on JSON files for me without this here:

```json
"[json]": {
  "editor.formatOnSave": true,
  "editor.defaultFormatter": "esbenp.prettier-vscode"
},
```

Does it work for you with just the `[javascript]` entry?

Is there a way to specify multiple languages so we don't have to repeat the same settings for `[css]` etc.?

Reply:
On the 2nd note, I tried 
```json
  "[css, javascript, json, html]": {
    "editor.formatOnSave": true,
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
```
 but it doesn't work

Reply:
💥 https://github.com/Microsoft/vscode/issues/51935#issuecomment-970436530

Reply:
_yesterday_

Reply:
Dangggg!